### PR TITLE
[Release notes] GraFx Studio InDesign plugin

### PR DESCRIPTION
**Pull request by jeroencosyn**

### CODE REVIEW

File: docs/release-notes/releaseposts/2024122001.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+draft: false
+date: 2024-12-20
+---
+
+# GraFx Studio - InDesign plugin
```
Comment (brapoprod) [inline docs/release-notes/releaseposts/2024122001.md:6]:
```suggestion
# GraFx Studio - Adobe® InDesign® plugin
```

File: docs/release-notes/releaseposts/2024122001.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+![rn_icon](/assets/icon-GraFx-Studio.svg)
+
+Looking to scale up production using your InDesign documents? With the new Adobe® InDesign® plugin, now available as [experimental](/release-notes/experimental/), you can seamlessly convert them into GraFx Studio templates.
```
Comment (brapoprod) [inline docs/release-notes/releaseposts/2024122001.md:10]:
```suggestion
Looking to scale up production using your Adobe® InDesign® documents? With the new Adobe® InDesign® plugin, now available as [experimental](/release-notes/experimental/), you can seamlessly convert them into GraFx Studio templates.
```

File: docs/release-notes/releaseposts/2024122001.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+<!-- more -->
+
+### Key Features:
+
+- **Export to GraFx Studio**:
+  Use the "GraFx Studio Exporter" in InDesign® to convert your designs to GraFx Studio packages.
```
Comment (brapoprod) [inline docs/release-notes/releaseposts/2024122001.md:17]:
```suggestion
  Use the "GraFx Studio Exporter" in Adobe® InDesign® to convert your designs to GraFx Studio packages.
```

**Review by BramVerniest — APPROVED** [review]
LPGTM

**Pull request merged**